language/stdlib/modules/validator_set.mvir

```diff
@@ -58,18 +58,21 @@ module ValidatorSet {
         return;
     }
 
+    // FIXME: The order of the last 2 arguments should be swapped to match the order
```

Comment:
This code will be refactored shortly so don't worry about it too much now.

Reply:
OK. I'll leave the FIXME for now so that we'll remember to clean this up when refactoring it.

Reply:
Nevermind. It has already been refactored and the comment is no longer relevant.